Makefile

```diff
@@ -316,7 +316,7 @@ else
 endif
 
 .PHONY: ci-manifest-build
-ci-manifest-build: manifest-build ## Build CI manifest
+ci-manifest-build: ## Build CI manifest
```

Comment:
do we need to do the same for all other repos too ?

Reply:
:heavy_check_mark: yes !